.gitattributes

```diff
@@ -0,0 +1 @@
+naksu2_*.deb filter=lfs diff=lfs merge=lfs -text
```

Comment:
Pitäisikö tän peräti olla `debs/*`?

Reply:
En halunnut koskea usb monsterin .deb:iin niin laitoin näin ja se on myös sinänsä turha olla tuolla LFS:ssä koska se on jotain 195kb iso

Reply:
Käy. Pidetään vain mielessä jos tulevaisuudessa halutaan tehdä näin